OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -502,7 +502,12 @@ Action InitInputPanel(Widget panel)
 					{ "ReplaySpeedSlowKey", "Slow speed" },
 					{ "ReplaySpeedRegularKey", "Regular speed" },
 					{ "ReplaySpeedFastKey", "Fast speed" },
-					{ "ReplaySpeedMaxKey", "Maximum speed" }
+					{ "ReplaySpeedMaxKey", "Maximum speed" },
+					{ "StatisticsBasicKey", "Basic Statistics" },
```

Comment:
Please use lower case "s" for consistency with (most of) the other definitions.
Don't worry about the other ones (e.g. "All Players"), I'll fix those in the pr that pulls these out to the yaml.